avocado/plugins/replay.py

```diff
@@ -263,15 +263,15 @@ def run(self, config):
             LOG_UI.warn("Ignoring variants from source job with "
                         "--replay-ignore.")
         else:
-            variants = jobdata.retrieve_variants(resultsdir)
+            variants = jobdata.get_variants_path(resultsdir)
             if variants is None:
                 LOG_UI.error('Source job variants data not found. Aborting.')
                 sys.exit(exit_codes.AVOCADO_FAIL)
             else:
-                LOG_UI.warning("Using src job Mux data only, use "
+                LOG_UI.warning("Using src job variants data only, use "
                                "`--replay-ignore variants` to override "
                                "them.")
-                config["avocado_variants"] = variants
+                config["json_variants_load"] = variants
```

Comment:
Maybe I'm missing something, but shouldn't this be registered before in a namespace format?

Reply:
You're right, thanks for spotting this!

I assumed this, at this stage, was effective because no sides effects could be seen, but it's not.  I'm sending a v2 with the needed changes.